ct-app/core/components/hoprd_api.py

```
@@ -1,256 +1,178 @@
     def log_prefix(cls) -> str:
         return "api"
 
     async def __call(
         self,
-        obj: Callable[..., object],
-        method: str,
-        *args,
-        **kwargs,
+        method: Method,
+        endpoint: str,
+        data: dict = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)